arangodb-net-standard/ViewApi/Models/ViewConsolidationPolicy.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace ArangoDBNetStandard.ViewApi.Models
+{
+    /// <summary>
+    /// The consolidation policy to apply
+    /// for selecting which segments should be merged.
+    /// Read more about this in the documentation.
+    /// </summary>
+    public class ViewConsolidationPolicy
+    {
+        /// <summary>
+        /// The type of consolidation policy.
+        /// 1) "tier" (default): consolidate 
+        /// based on segment byte size and live
+        /// document count as dictated by the 
+        /// customization attributes. 
+        /// If this type is used, then below 
+        /// <see cref="SegmentsBytesFloor"/>, 
+        /// <see cref="SegmentsBytesMax"/>,
+        /// <see cref="SegmentsMax"/>, 
+        /// <see cref="SegmentsMin"/> and 
+        /// <see cref="MinScore"/> are specified.
+        /// 2) "bytes_accum": consolidate only
+        /// if the sum of all candidate segment
+        /// byte size is less than the total 
+        /// segment byte size multiplied by 
+        /// the <see cref="ViewConsolidationPolicy.Threshold"/>.
+        /// </summary>
+        public string Type  { get; set; }
```

Comment:
I think this is a very good idea. I've made the changes to the Views API and will be making the changes to Analyzers soon. I prefer to keep them as constants within the same class, otherwise, we could run into trouble with naming :-)